### DISCUSSION TITLE
Just Completed first assignment from github classroom [Wrong due date]

### DISCUSSION THREAD

**Discussion by Sandeep-source** [skills/.github#133] [General]
I just completed my first assignment from github classroom.

This was a great ❤ experience. The grading system was pretty impressive with automated workflows.

The interesting part was the due date 🤔

![image](https://user-images.githubusercontent.com/61611561/210149643-fec37fd9-a2a1-4125-8856-5bf9faf00f68.png)

I got this assignment today [ 31st Dec 2022 ] but due date was 1st Dec 2022.

Is this an error or I got anything wrong?

What you think ?

**Comment by divais** [comment]
Thank you for sharing, we are looking into it.

**Comment by ericdrosado** [comment]
👋 Hi @Sandeep-source! We've updated the assignment. You should now have a proper due date. :octocat:

**Comment by Sandeep-source** [comment]
Thanks, @ericdrosado, and @divais for taking on the issue, it seems fixed.
